### PR TITLE
[ERR] Header | Verify notifications icon | AT_01.10_001| <Header> Verify Notifications icon

### PR DESCRIPTION
**Pull request by shablouskaya**
https://trello.com/c/NOR6ESj4/836-err-header-verify-notifications-icon

https://trello.com/c/pPGBMlwj/1208-rf-err-header-verify-notifications-icon-at0110001-header-verify-notifications-icon

### CODE REVIEW

File: cypress/e2e/verifyNotifacationsBell.cy.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 /// <reference types="cypress"/>
 
 describe('Header | Notifications icon', () => {
-    it.skip('Verify notifications icon', function () {
+    it('AT_01.10_001| <Header> Verify Notifications icon', function () {
+        cy.get('a[href="/manage"]').click()
+        cy.get('.jenkins-app-bar__content').should('have.text', 'Manage Jenkins')
```
Comment (DariaMas7) [inline cypress/e2e/verifyNotifacationsBell.cy.js:6]: 'Manage Jenkins' please move to fixtures

File: cypress/e2e/verifyNotifacationsBell.cy.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+        cy.get('.jenkins-breadcrumbs__list-item button[class="jenkins-menu-dropdown-chevron"]').realHover().realClick()
+        cy.get('.jenkins_ver a')
+          .should('exist')
+          .and('be.visible')
+          .and('have.text','Jenkins 2.387.2')
```
Comment (DariaMas7) [inline cypress/e2e/verifyNotifacationsBell.cy.js:11]: please move 'Jenkins 2.387.2' to fixtures

File: cypress/e2e/verifyNotifacationsBell.cy.js [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
         cy.get('#visible-am-button').click()
         cy.get('#visible-am-list a[href="/manage"]').should('have.text', 'Manage Jenkins')
```
Comment (DariaMas7) [inline cypress/e2e/verifyNotifacationsBell.cy.js:13]: 'Manage Jenkins' to fixtures

**Review by DariaMas7 — COMMENTED** [review]
While working on ERR card you should create RF card in order to make changes to you code
RF card should go as a name in your commit, and you should attach link to trello RF card
In trello you should do ERR <-> RF (attach cards to each other)

**Review by SiarheiMSV — APPROVED** [review]

**Review by IharStankevich — APPROVED** [review]

**Comment by shablouskaya** [comment]
> While working on ERR card you should create RF card in order to make changes to you code RF card should go as a name in your commit, and you should attach link to trello RF card In trello you should do ERR <-> RF (attach cards to each other)

Created RF card

**Pull request merged**